giraffez/commandline.py

```diff
@@ -415,6 +418,9 @@ def run(self, args):
             show_warning(("USING MLOAD TO INSERT LESS THAN {} ROWS IS NOT RECOMMENDED - USE LOAD "
                 "INSTEAD!").format(MLOAD_THRESHOLD), UserWarning)
 
+        if args.delimiter is None:
```

Comment:
Was this behavior not working previously, or was the delimiter inferring moved? Should this be behind a check for `encoding == "archive"`?